fenix/views/clients_counts_pop_method1.view.lkml

```diff
@@ -0,0 +1,83 @@
+include: "//looker-hub/fenix/views/client_counts.view.lkml"
+
+
+view: clients_counts_pop_method1 {
+  extends: [client_counts]
+  dimension_group: first_seen_pop {
+    # sql: ${TABLE}.first_seen_date ;;
+    type: time
+    view_label: "_PoP"
+    timeframes: [
+      raw,
```

Comment:
is `raw` just original value? We can probably remove it? `date` is probably lowest granularity we'd want to look at?